.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "asciidoc.antora.enableAntoraSupport": true
+}
```

Comment:
no new line

Reply:
sorry, this is unintended, let me remove. also, @ibuziuk i am unable to login to che-dogfooding.apps... it gives me authentication error, is it expected or something ?